src/main/scala/com/gu/crossword/crosswords/APIChecker.scala

```diff
@@ -23,23 +30,34 @@ trait APIChecker {
 
     import scala.concurrent.ExecutionContext.Implicits.global
 
-    val h = new Http()
+    def check200(req: Request): Future[Boolean] = {
+      val promise = Promise[Boolean]()
 
-    def check200(request: Req) = h(request).map(resp => {
-      if (resp.getStatusCode == 200) {
-        true
-      } else {
-        // we don't want to log live capi failures, as these are expected for all future crosswords and make the logging messy
-        if (!request.url.contains("guardianapis.com") || request.url.contains("preview")) {
-          println(s"Didn't get 200 response from ${request.url}. Actual response: ${resp.getStatusText} ${resp.getStatusCode}")
-          println(s"Response body: ${resp.getResponseBody}")
+      http.newCall(req).enqueue(new Callback() {
+        override def onFailure(call: Call, e: IOException): Unit = promise.failure(e)
+        override def onResponse(call: Call, resp: Response): Unit = {
+          if (resp.code == 200) {
+            promise.success(true)
+          } else {
+            // we don't want to log live capi failures, as these are expected for all future crosswords and make the logging messy
+            if (!req.url.host.contains("guardianapis.com") || req.url.host.contains("preview")) {
```

Comment:
will this check still work with the new api gateway url?

Reply:
yep, the urls are similar